.github/workflows/main.yml

```diff
@@ -50,7 +51,32 @@ jobs:
           flake8 . --exit-zero --max-complexity=6
 
       - name: Upload python artifacts
+        if: ${{ github.ref_name == 'main' }}
         uses: actions/upload-artifact@v4
         with:
           name: python-artifacts
           path: .
+
+docker-ci:
```

Comment:
Incorrect format of YAML film. For this reason, pipeline failed
![image](https://github.com/mate-academy/devops_todolist_cicd_task_3_docker_ci/assets/87720583/d15a308c-a9c9-405c-991c-c80733b72da0)

```suggestion
   docker-ci:
```